src/content/sessions/2023-04-07.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Migrating Vite to Perla
+preview: Migrating Fantomas Tools to Perla!
+isDraft: false
+date: 2023-04-07
+slug: "2023/03/24"
+champion: Angel D. Munoz
+company: Encora
+repository: https://github.com/AngelMunoz/Perla
+---
+
+## Problem
+
+[Fantomas tools](https://fsprojects.github.io/fantomas-tools/#/) is a collection of great utilities to use when working on [Fantomas](https://fsprojects.github.io/fantomas/) or the [F# untyped AST](https://fsharp.github.io/fsharp-compiler-docs/reference/fsharp-compiler-syntax.html).  
+The frontend using [Fable](https://fable.io/) &amp; [React](https://react.dev/) and the backend are a bunch of [AWS Lambda](https://aws.amazon.com/blogs/developer/f-tooling-support-for-aws-lambda/) functions.
+
+These Lambda functions are an interesting way to keep the cost low, but they can take a while to start up.  
+So, the advice from the maintainers is to run things locally to speed up the experience.
+
+<img src="/images/sessions/fantomas-tool.png" alt="Fantomas Tools Welcome message" class="img-fluid mb-4" />
+
+However, to this you need to install a bunch of dependencies, NodeJS among them, and it can be a bit of a pain to get everything working.
+
+## Expected behavior
+
+It would be nice if we could swap out the `Node.JS` bits and replace them with `Perla` to make the experience a bit more pleasant.  
+[Perla](https://perla-docs.web.app/) is a modern development web server for front-end development written in `dotnet`.  
+Because Perla is shipped as a `dotnet` tool, you only need to `dotnet` SDK to get started.
```

Comment:
```suggestion
Because Perla is shipped as a `dotnet` tool, you only need to `dotnet` SDK to get started, the binaries are self-contained so this also removes the dotnet requirement for cases where you work in constrained environments.  
```